Code/GraphMol/ChemReactions/ReactionParser.h

```diff
@@ -79,7 +79,7 @@ ChemicalReaction *RxnDataStreamToChemicalReaction(std::istream &rxnStream,
 ChemicalReaction *RxnSmartsToChemicalReaction(
     const std::string &text,
     std::map<std::string, std::string> *replacements = 0,
-    bool useSmiles = false);
+    bool useSmiles = false, bool sanitize=false);
```

Comment:
the `sanitize` argument here should also be renamed to `sanitizeComponents`
[Required]